docs/revsets.md

```diff
@@ -362,6 +368,18 @@ given [string pattern](#string-patterns).
     * `roots(E|A)` ⇒ `{A}`
     * `roots(A)` ⇒ `{A}`
 
+    **function** `fork_point()`
+
+    * `fork_point(E|D)` ⇒ `{A}`
+    * `fork_point(E|C)` ⇒ `{A}`
+    * `fork_point(E|B)` ⇒ `{B}`
+    * `fork_point(E|A)` ⇒ `{A}`
+    * `fork_point(D|C)` ⇒ `{C}`
+    * `fork_point(D|B)` ⇒ `{A}`
+    * `fork_point(B|C)` ⇒ `{A}`
+    * `fork_point(A)` ⇒ `{A}`
+    * `fork_point()` ⇒ `{}`
```

Comment:
nit: maybe `fork_point(none())` so users don't think it can be called with no arguments